sjb/config/test_cases/test_pull_request_crio_e2e_features_rhel.yml

```diff
@@ -0,0 +1,36 @@
+---
+parent: 'common/test_cases/crio.yml'
+overrides:
+  provision:
+    os: "rhel"
+    stage: "crio"
+    provider: "aws"
+extensions:
+  actions:
+  - type: "forward_parameters"
+    parameters:
+      - PULL_NUMBER
+      - PULL_BASE_SHA
+      - PULL_PULL_SHA
+  - type: "script"
+    title: "clone cri-o and check out the correct refs"
+    script: |-
+      cd /go/src/github.com/kubernetes-sigs/cri-o
+      git fetch origin
+      git checkout master
+      git branch -D target || true
+      git branch -D pr || true
+      git branch target "${PULL_BASE_SHA}"
+      git checkout target
+      git fetch origin "pull/${PULL_NUMBER}/head:pr"
+      git merge "${PULL_PULL_SHA}"
+  - type: "script"
```

Comment:
@stevekuznetsov I'm wondering if it possible to override the actions above so I can pick them up directly from the base e2e script and just override this script for the new jobs

Reply:
you could make a base job that only does the clone, but i don't think it's worth it -- that code has never changed, right?

Reply:
yeah it never changed indeed, was just wondering though :)